packages/shared/src/permissions/permissions.utils.ts

```diff
@@ -2,8 +2,266 @@ import {
   Permission,
   UserPermissions,
   MemberRole,
+  PermissionsObject,
+  OrganizationInterface,
+  DefaultMemberRole,
+  Role,
 } from "back-end/types/organization";
 
+export const POLICIES = [
+  "ReadData",
+  "Comments",
+  "FeaturesFullAccess",
+  "ArchetypesFullAccess",
+  "FeaturesBypassApprovals",
+  "ExperimentsFullAccess",
+  "VisualEditorFullAccess",
+  "SuperDeleteReports",
+  "DataSourcesFullAccess",
+  "DataSourceConfiguration",
+  "RunQueries",
+  "MetricsFullAccess",
+  "FactTablesFullAccess",
+  "FactMetricsFiltersFullAccess",
+  "DimensionsFullAccess",
+  "SegmentsFullAccess",
+  "IdeasFullAccess",
+  "PresentationsFullAccess",
+  "SDKPayloadPublish",
+  "SDKConnectionsFullAccess",
+  "AttributesFullAccess",
+  "EnvironmentsFullAccess",
+  "NamespacesFullAccess",
+  "SavedGroupsFullAccess",
+  "GeneralSettingsFullAccess",
+  "NorthStarMetricFullAccess",
+  "TeamManagementFullAccess",
+  "ProjectsFullAccess",
+  "TagsFullAccess",
+  "APIKeysFullAccess",
+  "IntegrationsFullAccess",
+  "EventWebhooksFullAccess",
+  "BillingFullAccess",
+  "AuditLogsFullAccess",
+] as const;
+
+export type Policy = typeof POLICIES[number];
+
+export const POLICY_PERMISSION_MAP: Record<Policy, Permission[]> = {
+  ReadData: ["readData"],
+  Comments: ["readData", "addComments"],
+  FeaturesFullAccess: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "canReview",
+  ],
+  ArchetypesFullAccess: ["readData", "manageArchetype"],
+  FeaturesBypassApprovals: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "canReview",
+    "bypassApprovalChecks",
+  ],
+  ExperimentsFullAccess: ["readData", "createAnalyses"],
+  VisualEditorFullAccess: ["readData", "manageVisualChanges"],
+  SuperDeleteReports: ["readData", "superDeleteReport"],
+  DataSourcesFullAccess: [
+    "readData",
+    "createDatasources",
+    "editDatasourceSettings",
+  ],
+  DataSourceConfiguration: ["readData", "editDatasourceSettings"],
+  RunQueries: ["readData", "runQueries"],
+  MetricsFullAccess: ["readData", "createMetrics"],
+  // TODO: Fix fact permissions
+  FactTablesFullAccess: ["readData", "manageFactTables", "createMetrics"],
```

Comment:
@jdorn For this - it looks like we'll need to create new permissions to handle letting users manage Fact Metrics and Fact Filters. Both of those currently use `createMetrics` under the hood.

Given that, what are your thoughts on adding the following permissions:
- createFactMetrics, updateFactMetrics, deleteFactMetrics
- createFactFilters, updateFactFilters, deleteFactFilters

Note: In doing so, we can update then update their associated Permissions Class method.

Reply:
E.G. We would then have FactTablesFullAccess, FactMetricsFullAccess, and FactFiltersFullAccess.

```ts
FactTablesFullAccess: ["readData", "manageFactTables", "manageFactMetrics", "manageFactFilters"],
FactMetricsFullAccess: ["readData", "manageFactMetrics"],
FactMetricsFullAccess: ["readData", "manageFactFilters"],
```

Reply:
I think I want to keep metrics and filters together for now.  There are some upcoming projects that are going to blur the line between them